Update OAuth2 client

brianhelba opened the pull request:
The library `@isic/client` is being retired, in favor of directly using the `@resonant/oauth-client` library which it depended on.

brianhelba commented:
@volishevko Please take a look. We need to update this library in order to make some internal changes on the server.

volishevko commented:
> @volishevko Please take a look. We need to update this library in order to make some internal changes on the server.

There is an error on logout.
[oauth-client.isic-display-ui.pages.dev-1738675378742.log](https://github.com/user-attachments/files/18657683/oauth-client.isic-display-ui.pages.dev-1738675378742.log)

brianhelba commented:
@volishevko Good find, thanks. I've fixed the error upstream and updated this to use a newer version of the library.

Let me know if it's working for you now too.

volishevko reviewed (approved):
Everything looks good to me. Approved.
